### PR TITLE
create support issue template

### PR DESCRIPTION
**Pull request by jsanda**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/support.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- [ ] How to perform a particular operation
+- [ ] Cassandra-related question
+- [ ] Monitoring-related question
+- [ ] Repair-related question
+- [ ] Backup/restore-related question
+- [ ] Open question
+
+**What did you do?**
+
+**Did you expect to see some different?**
```
Comment (burmanm) [inline .github/ISSUE_TEMPLATE/support.md:33]: See some difference or see something different?

File: .github/ISSUE_TEMPLATE/support.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+<!--
+    You can try a jsonpath query with kubectl like this to get the version:
+
+        kubectl get deployment <reaper-operator-deployment> \
+            -o jsonpath='{.spec.template.spec.containe[0].image}'
+-->
+
+<!--
+Please provide the follow info if you deployed reaper-operator via the
```
Comment (burmanm) [inline .github/ISSUE_TEMPLATE/support.md:48]: following info?

**Review by jdonenine — CHANGES_REQUESTED** [review]
Just a couple of small typos.

**Review by jdonenine — APPROVED** [review]
I pushed a commit with the fixes for the minor typos.  Good to go.

**Pull request merged**